unstructured/partition/pdf_image/analysis/layout_dump.py

```diff
@@ -0,0 +1,67 @@
+import json
+from abc import ABC, abstractmethod
+from pathlib import Path
+
+from unstructured_inference.inference.layout import DocumentLayout
+
+from unstructured.partition.pdf_image.analysis.processor import AnalysisProcessor
+
+
+class LayoutDumper(ABC):
+    layout_source: str = "unknown"
+
+    @abstractmethod
+    def dump(self) -> dict:
+        """Transforms the results to a dict convertible structured formats like JSON or YAML"""
+
+
+def extract_layout_info(layout: DocumentLayout):
+    pages = []
+
+    for page in layout.pages:
+        size = [page.image_metadata.get("width"), page.image_metadata.get("height")]
+        elements = []
+        for element in page.elements:
+            bbox = element.bbox
+            elements.append(
+                {
+                    "bbox": [bbox.x1, bbox.y1, bbox.x2, bbox.y2],
+                    "type": element.type,
```

Comment:
those "types" I found in dumps are not compatible with `YOLO_SG_LABEL_MAP`, which I need to evaluate OD model,
I also need list of `YOLO_SG_LABEL_MAP` in json (as "categories" eg.) as I should not import it from _unstructured_inference_prop/object_detection/yolox_super_gradients.py_ in evaluate

Reply:
Added in the last commit.